example/eternity.bora.sh/config.yaml

```diff
@@ -65,3 +65,6 @@ menu:
     - name: about
       url: /about/
       weight: 5
+    - name: tags
```

Comment:
I think example projects doesn't need to have a tags in the menu. Because used tags count is so low and almost all used tags are in the menu. There's 2 tags not in the menu, so I think we can keep tags page as hidden, from menu.

Reply:
I've removed the entry into the menu.